api/src/main/java/nl/fhict/happynews/api/controller/PostController.java

```diff
@@ -38,6 +38,17 @@
         }
     }
 
+
+    /**
+     * Handles a GET request by returning posts in a paginated format.
+     * @param pageable the page and page size
+     * @return A Page with Post information
+     */
+    @RequestMapping(value = "/postpage", method = RequestMethod.GET, produces = "application/json")
```

Comment:
I think it would make sense to remove the `/post` API, and move this to `/post`.